src/components/NetworkStatusService.js

```diff
@@ -60,13 +58,8 @@ goog.require('ga_waitcursor_service');
     var count = 0;
     var promise;
     this.$get = function($document, $rootScope, $timeout, $window,
-        gaBrowserSniffer, gaGlobalOptions) {
-      if (!gaBrowserSniffer.mobile) {
-        return {
-          offline: false,
-          check: function() {}
-        };
-      }
+        gaGlobalOptions) {
+
```

Comment:
What are the consequences? Do desktop apps now poll the checker all the time?

Or only when offline is detected first?